src/commands/BulkCommand.ts

```diff
@@ -0,0 +1,82 @@
+import { Message, User } from 'discord.js';
+import PrefixCommand from './PrefixCommand';
+import MentionCommand from './MentionCommand';
+import ResolveRequestMessageTask from '../tasks/ResolveRequestMessageTask';
+import TaskScheduler from '../tasks/TaskScheduler';
+import emojiRegex = require( 'emoji-regex/text.js' );
+import { RequestsUtil } from '../util/RequestsUtil';
+import BotConfig from '../BotConfig';
+
+export default class BulkCommand extends PrefixCommand {
+	public readonly aliases = ['bulk', 'filter'];
+
+	public static currentBulkReactions: Map<User, Message[]>;
+
+	public async run( message: Message, args: string ): Promise<boolean> {
+		let rawEmoji: string;
+
+		if ( args.length ) {
+			const customEmoji = /^<a?:(\w+):(\d+)>/;
+			const unicodeEmoji = emojiRegex();
+
+			if ( customEmoji.test( args ) || unicodeEmoji.test( args ) ) {
+				rawEmoji = args;
+				const emojiMatch = customEmoji.exec( args );
+				if ( emojiMatch ) {
+					rawEmoji = emojiMatch[2];
+				}
```

Comment:
The logic here looks tricky. I'd suggest using

```suggestion
			if ( unicodeEmoji.test( args ) ) {
				rawEmoji = args;
			} else if ( customEmoji.test( args ) ) {
				rawEmoji = customEmoji.exec( args )[2];
```